cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -66,7 +67,7 @@
   private final LoadMonitor _loadMonitor;
   private final GoalOptimizer _goalOptimizer;
   private final ExecutorService _goalOptimizerExecutor;
-  private final Executor _executor;
+  private Executor _executor;
```

Comment:
It is not clear why we drop `final` here. Can we avoid this change?

Reply:
This is part of the previous hacky way to get around of the cycler dependency. You are right, this is not needed anymore!